File: client/src/models/Part.js
export class Part{
    constructor(data){
        this.id = data.id
        this.description = data.description
        this.partDescription = data.partDescription
        this.partNumber = data.partNumber
        this.quantity = data.quantity
        this.uom = data.uom
        this.creator = data.creator
        this.creatorId = data.creatorId
        this.orderId = data.orderId
        this.createdAt = new Date(data.createdAt).toLocaleTimeString()
        this.updatedAt = new Date(data.updatedAt).toLocaleTimeString()
    }
}

File: server/Models/Part.cs
namespace fishbowlManual.Models;
public class Part
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Description { get; set; }
    public string PartNumber { get; set; }
    public string PartDescription { get; set; }
    public int? Quantity { get; set; }
    public string UOM { get; set; }
    public int OrderId { get; set; }
    public Account Creator { get; set; }
    public string CreatorId { get; set; }
}